ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDOrderDirection.java

```diff
@@ -1,15 +1,39 @@
 package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
 import lombok.Builder;
 import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
 
 import java.time.LocalDate;
+import java.util.List;
 
 @Value
 @Builder
+@JsonTypeName(value = "OtherDirection")
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class CCDOrderDirection {
     private CCDOrderDirectionType extraOrderDirection;
-    private String otherDirection;
+    private String directionComment;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    @JsonSerialize(using = LocalDateSerializer.class)
     private LocalDate sendBy;
     private CCDDirectionPartyType forParty;
+    private List<CCDCollectionElement<String>> extraDocUploadList;
+    private List<CCDCollectionElement<String>> expertReport;
+
+    @JsonProperty("expertReport")
+    void setExpertReport(List<CCDCollectionElement<String>> expertReport) {
+        setExpertReport(expertReport);
+    }
+
+    @JsonProperty("expertReportPermissionStatementList")
```

Comment:
why getExpertReport need expertReportPermissionStatementList in getter and expertReport in setter?

Reply:
because we have `expertReportPermissionStatementList` in the template - so it needs to be serialised like that and we have `expertReport` in the definition file - so it needs to be deserialised like that :(

Reply:
Prefer to use the same on both places.